cyber-threat-coalition/src/cyber-threat-coalition.py

```diff
@@ -107,10 +108,12 @@ def fetch_and_send(self):
                         opencti_type = "ipv4-addr"
                     elif collection == "url":
                         opencti_type = "url"
+                        data = urllib.parse.quote(data)
```

Comment:
Using quote() without options will result in url encoding the ":" after the protocol. E.g:
data = "http://example.com/view?a'b" => `http%3A//example.com/view%3Fa%27b`

Don't know if that would affect the back-end (other functionality, interpretation from the stix schema etc).
```suggestion
                        data = urllib.parse.quote(data, "/:")
```
The above will convert the url like so: http://example.com/view%3Fa%27b, which i think may be more compatible. Only downside is that you consider ":" safe for the rest of the url.